lib/fuelsdk/objects.rb

```diff
@@ -36,21 +36,21 @@ def perform options
     module Rest
       module Read
         def get
-          client.rest_get id, Array.wrap(properties)
+          client.rest_get id, properties
         end
       end
 
       module CUD
         def post
-          client.rest_post id, Array.wrap(properties)
+          client.rest_post id, properties
         end
 
         def patch
-          client.rest_patch id, Array.wrap(properties)
+          client.rest_patch id, properties
         end
 
         def delete
-          client.rest_delete id, Array.wrap(properties)
```

Comment:
Ya, I'm a bit hazy on the details @leebs778  but it was bombing in the `rest` module, IIRC,  because it was not expecting an array.  Seems like the REST API support in this gem is pretty incomplete.